applications/GeoMechanicsApplication/tests/cpp_tests/test_Hencky_strain.cpp

```diff
@@ -0,0 +1,95 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+#include "testing/testing.h"
+#include "custom_utilities/math_utilities.hpp"
+#include "custom_utilities/stress_strain_utilities.hpp"
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CheckHenckyStrainIncompressibleUniaxialElongation, KratosGeoMechanicsFastSuite)
+{
+    double stretch = 2.0;
+    Matrix Fmatrix = ZeroMatrix(3,3);
+    Fmatrix(0,0) = stretch;
+    Fmatrix(1,1) = 1.0/std::sqrt(stretch);
+    Fmatrix(2,2) = Fmatrix(1,1);
+
+    Vector Evector = StressStrainUtilities::CalculateHenckyStrain(Fmatrix, 6);
+    KRATOS_EXPECT_DOUBLE_EQ(0.5*std::log(stretch*stretch), Evector[0]);
```

Comment:
It might be nice to add a comment here on why the HenckyStrain vector of this matrix has these results (to me it's not trivial. If it just follows by going through the calculation, then it's not necessary.

Reply:
added comment on fill of the Hencky strain